From ae091a7df2fb9e2a322da18205c0108406c682ba Mon Sep 17 00:00:00 2001
From: chdugw_auto <jkulkarni@maxlinear.com>
Date: Tue, 17 Aug 2021 17:14:26 +0300
Subject: [PATCH] Merge pull request #3478 in SW_UGW/linux from
 feature/PONRTSYS-8997-lgm-pon-multicast-support-master2 to master

* commit '356e1f3907b69afa0b495947045bd13f84394378':
  PONRTSYS-8997: URX mulitcast support
---
 drivers/net/ethernet/lantiq/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/lantiq/Makefile b/drivers/net/ethernet/lantiq/Makefile
index c89d8ee0e079..ad1b33b01263 100644
--- a/drivers/net/ethernet/lantiq/Makefile
+++ b/drivers/net/ethernet/lantiq/Makefile
@@ -1,6 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 #
 # Copyright (C) 2019 Intel Corporation.
+# Copyright (c) 2021 MaxLinear, Inc.
 obj-$(CONFIG_LANTIQ_VRX318) += ltq_vrx318.o
 obj-$(CONFIG_LANTIQ_ETH_FRAMEWORK) += lantiq_eth_framework.o
 obj-$(CONFIG_INTEL_XPCS) += xpcs/
@@ -21,4 +22,4 @@ obj-$(CONFIG_LTQ_TOE_DRIVER) += ltq_toe_drv.o
 obj-$(CONFIG_INTEL_MIB_COUNTERS) += ../../datapath/mib_counters/
 obj-$(CONFIG_PON_QOS) += ../../datapath/pon_qos/
 obj-$(CONFIG_DP_ETH_REINSERT) += ../../datapath/dp_eth_reinsert/
-obj-$(CONFIG_DP_MCAST_BRIDGE_FILTER) += ../../datapath/bridge_filter/
+obj-y += ../../datapath/bridge_filter/
